src/objects/GS_Admin_Checklist_Item__mdt.object

```diff
@@ -153,22 +157,22 @@
     </fields>
     <fields>
         <fullName>Secondary_Button_Label__c</fullName>
-        <description>The api name for the Secondary Button in a section item on the Get Started page.</description>
+        <description>The API name for Secondary Button label in a section item on the Get Started page. Must match an imported label in the gsLabelMapper LWC.</description>
         <externalId>false</externalId>
         <fieldManageability>DeveloperControlled</fieldManageability>
-        <inlineHelpText>The api name for the Secondary Button in a section item on the Get Started page.</inlineHelpText>
-        <label>Secondary Button Label</label>
+        <inlineHelpText>The API name for Secondary Button label in a section item on the Get Started page. Must match an imported label in the gsLabelMapper LWC.</inlineHelpText>
```

Comment:
Can you please add "the" before Secondary so that this read: The "API name for the Secondary Button..."

Please make this change in the inlineHelpText too.